x-pack/plugins/security_solution/common/detection_engine/build_exceptions_filter.test.ts

```diff
@@ -0,0 +1,1099 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  getEntryMatchMock,
+  getEntryMatchExcludeMock,
+} from '../../../lists/common/schemas/types/entry_match.mock';
+import {
+  getEntryMatchAnyMock,
+  getEntryMatchAnyExcludeMock,
+} from '../../../lists/common/schemas/types/entry_match_any.mock';
+import {
+  getEntryExistsMock,
+  getEntryExistsExcludedMock,
+} from '../../../lists/common/schemas/types/entry_exists.mock';
+import {
+  getEntryNestedMock,
+  getEntryNestedExcludeMock,
+  getEntryNestedMixedEntries,
+} from '../../../lists/common/schemas/types/entry_nested.mock';
+import {
+  getExceptionListItemSchemaMock,
+  getExceptionListItemSchemaXMock,
+} from '../../../lists/common/schemas/response/exception_list_item_schema.mock';
+
+import {
+  buildExceptionItemFilter,
+  buildExclusionClause,
+  buildExistsClause,
+  buildMatchAnyClause,
+  buildMatchClause,
+  buildNestedClause,
+  createOrClauses,
+  chunkExceptions,
+  buildExceptionFilter,
+  ExceptionItemSansLargeValueLIsts,
```

Comment:
You have a weird capital `I` here, I think you want to rename this as:

```ts
ExceptionItemSansLargeValueLists
```

Reply:
It's what all the cool kids are doing Frank... 😉 Fixed!